fetch_teleop/src/joystick_teleop.cpp

```diff
@@ -55,6 +57,22 @@ double integrate(double desired, double present, double max_rate, double dt)
     return std::max(desired, present - max_rate * dt);
 }
 
+double average(std::list<double>& prev_values, float avg, double next_value) 
```

Comment:
`avg` can actually be an `int` because you are only comparing it with the size of the list.